app/assets/javascripts/base.js.erb

```diff
@@ -684,7 +688,7 @@ var instantClick
     apiKey: "<%= ApplicationConfig['HONEYBADGER_JS_API_KEY'] %>",
     environment: "<%= Rails.env %>",
     revision: "<%= ApplicationConfig['HEROKU_SLUG_COMMIT'] %>",
-    ignorePatterns: [/ResizeObserver/i, /MetaMask detected another web3/i],
+    ignorePatterns: [/ResizeObserver/i, /MetaMask/i],
```

Comment:
@Zhao-Andy I think quoting this change is more appropriate. I'm loosening up the regex constraint so the ignore-pattern can be applied to more errors.